cage.c

```diff
@@ -101,14 +103,23 @@ parse_args(struct cg_server *server, int argc, char *argv[])
 {
 	int c;
 #ifdef DEBUG
-	while ((c = getopt(argc, argv, "dDh")) != -1) {
+	while ((c = getopt(argc, argv, "drfDh")) != -1) {
 #else
-	while ((c = getopt(argc, argv, "dh")) != -1) {
+	while ((c = getopt(argc, argv, "drfh")) != -1) {
 #endif
 		switch (c) {
 		case 'd':
 			server->xdg_decoration = true;
 			break;
+		case 'r':
+			server->output_rotation++;
+			if (server->output_rotation > WL_OUTPUT_TRANSFORM_270) {
+				server->output_rotation = WL_OUTPUT_TRANSFORM_NORMAL;
+			}
+			break;
+		case 'f':
```

Comment:
Honest question: what is the use case for flipping the output?

Reply:
I honestly don't know either .. maybe some weird devices need this? Given that it's part of the transform enum I thought I would make a flag for it. I can remove it if you don't feel like it deserves a dedicated flag.

Reply:
Also, is passing the rotation flag multiple times OK with you? I wanted to avoid doing more parsing and went for the lazy way, but there might be a more elegant way.

Reply:
Please remove the flag if there is no good use case for it. It'll be easy to add back if the feature is ever requested.

Passing `-r` multiple times is OK with me. I think it's rather elegant in a way :)